database/spanner/spanner_test.go

```diff
@@ -151,6 +151,14 @@ func TestCleanStatements(t *testing.T) {
 			SET OPTIONS (allow_commit_timestamp=true);`,
 			expected: []string{"ALTER TABLE users ALTER COLUMN created SET OPTIONS (allow_commit_timestamp = true)"},
 		},
+		{
+			name: "column with NUMERIC type",
```

Comment:
This only tests that `cleanStatements()` passes. It doesn't test running the statement against the emulator. Was this your intention?

Reply:
Whoops, just updated where the last spanner upgrade PR did https://github.com/golang-migrate/migrate/pull/427 -- will add a migration to test this case as well. Let me know if there's anywhere else that should be updated!

Reply:
If DDL parsing is all you want to test, then this is fine. But if you want to test table creation, then this doesn't test that. I'm fine either way.

Reply:
Hi @dhui not sure I understand, when you say 'these examples' are you saying the tests are good as is, or that I should remove the migrations and add a new test function? If you have an example of what you're referring to that would be great